npc/re/custom/lasagna/lasagna_npcs_2.txt

```diff
@@ -0,0 +1,2065 @@
+			mes "[Shaman meow]";
+			mes "^0000cd20 basils and 5 basil locks^000000. Same as the first. The most basic is the most useful.";
+			next;
+			mes "[Shaman meow]";
+			mes "I pray for your peace and safety. Tell the boatswain to come back anytime he changes his mind.";
+			erasequest 14559;// Charm Effect Duration
+			setquest 14562;// The basic are the best
+			close;
+		}
+		if (isbegin_quest(14562) == 1) {
+			mes "[Shaman meow]";
+			mes "Today's talisman is ^0000cd20 basils and 5 basil locks^000000. Same as the first. The most basic is the most useful.";
+			next;
+			mes "[Shaman meow]";
+			mes "I pray for your peace and safety. Tell the boatswain to come back anytime he changes his mind.";
+			close;
+		}
+		// continue
+	}
+	mes "[Shaman meow]";
+	mes "You want to know the future? The past? If you know where you're standing, then you can see the past, future, everything.";
+	next;
+	mes "[Shaman meow]";
+	mes "If you need to curse someone, break a curse, or need a bit of luck, then come see me.";
+	close;
+}
+lasagna,174,160,3	script	Vigilante Guide#lasa	4_DR_SOLDIER,5,5,{
+	mes "[Vigilante Guide]";
+	mes "Hello.";
+	mes "The weather is good today, isn't it?";
+	mes "What can I do for you?";
+	next;
+	switch( select( "Talk about the weather.", "Ask about the Vigilante.", "Ask about Lasagna.", "End dialogue." ) ) {
+	case 1:
+		mes "[Vigilante Guide]";
+		mes "In Lasagna, a fresh sea breeze always tickle your whiskers.";
+		mes "And thanks to this moist air, our cheese ripen deliciously.";
+		close;
+	case 2:
+		if (lasagna_quest[1] < 2) {
+			if (lasagna_quest[1] == 0) {
+				mes "[Vigilante Guide]";
+				mes "Oh! You haven't met our chief yet?";
+				next;
+			}
+			mes "[Vigilante Guide]";
+			mes "Hmm...";
+			mes "I think it'd be best for you to see the chief first.";
+			next;
+			mes "[Vigilante Guide]";
+			mes "He should be at the Adventurers' Club around now...";
+			mes "The Adventurers' Club is to the east from here, so it shouldn't be difficult to find.";
+			if (isbegin_quest(11435) == 0)
+				setquest 11435;// Vigilante Corp
+			navigateto "lasagna",235,198, NAV_KAFRA_ONLY, 0;
+			lasagna_quest[1] = 1;
+			close;
+		}
+		mes "[Vigilante Guide]";
+		mes "The Vigilante Corp not only maintain public order in the town, but also play the main role in running the town.";
+		mes "Maybe our chief should be called the mayor instead.";
+		close;
+	case 3:
+		mes "[Vigilante Guide]";
+		mes "Lasagna is a port town located at the end of the Far-Star continent.";
+		mes "it's a small but a thriving town thanks to the efforts of Chief Peperoncino, Captain Gamberi, and Captain Tomatomas!";
+		close;
+	case 4:
+		mes "[Vigilante Guide]";
+		mes "Good bye.";
+		close;
+	}
+OnTouch:
+	npctalk "Vigilante Guide: Welcome to the port town Lasagna.", "", bc_self;
+	npctalk "Vigilante Guide: If you're ever in need of help, please visit the Vigilante Corp at any time.", "", bc_self;
+	end;
+OnInit:
+	questinfo 11435, QTYPE_QUEST;
```

Comment:
this and another line (i will show it below) provoke errors at clientside like on screenshot, and everytime (every new restart) new version of the error. 

Error (every  server restart the error is different (quest_3242, quest_204, quest_52, quest_18, etc)
![](https://i.imgur.com/k6bbmBM.png)


Then i have replace "missing resource" via red dot on mininmap, and found a reason of the error. This is this NNPC & this code line

![](https://i.imgur.com/IR2IDBx.png)